turbinia/processors/mount_local.py

```diff
@@ -21,14 +21,17 @@
 import subprocess
 import tempfile
 import time
-
+from prometheus_client import Gauge
 from turbinia import config
 from turbinia import TurbiniaException
 
 log = logging.getLogger('turbinia')
 
 RETRY_MAX = 10
 
+turbinia_failed_detach_disk = Gauge(
```

Comment:
Can we also add these to the monitoring dashboards?

Reply:
Yes but they need to be done on the dashboard itself

Reply:
We talked out of band, and alimez is going to add these to the dashboard directly and then export and sync the resulting .json file into the repo.